README.md

```diff
@@ -60,28 +60,37 @@ The following is a talk at Spark+AI Summit 2018 about MAX that includes a short
 [![Watch the Video](https://i.vimeocdn.com/video/707093466.jpg)](https://vimeo.com/274936527)
 
 # Steps
-<!--
-Use the ``Deploy to IBM Cloud`` button **OR** run locally.
+
+Deploy to IBM Cloud **OR** run locally.
 
 ## Deploy to IBM Cloud
 
-[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/MAX-Image-Caption-Generator-Web-App)
+### Deploy the Model
 
-1. Press the above ``Deploy to IBM Cloud`` button and then click on ``Deploy``. If you do not have an IBM Cloud account
-yet, you will need to create one.
+Follow the [Deploy the Model Doc]() to deploy the model to IBM Cloud.
```

Comment:
this link needs to be filled.

Reply:
Yeah that's a placeholder til Daniel finishes his Doc to link to, I'm ok waiting to merge this until he's done.